packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -23,7 +23,6 @@ const StyledTooltip = styled.div`
   /* Overriding the default popover styles */
   display: none;
   &[popover] {
-    position: absolute;
```

Comment:
This change was introduced to in https://github.com/primer/react/pull/4250 to fix an issue on the Tooltip V2 react docs but it ended up causing a problem at dotcom (specifically memex), flickering and shifting when the tooltip appears. Without this line, everything except the docs is working fine. I think the issue is not on the tooltip but on docs, maybe? I'd like to take this change back to unblock the version upgrade and can look into further later. Though, all react docs is going to be migrated over to the new docs and the way that examples are rendered differently (from stories) on the new site so I'll see if the same issue persistent on the new docs first.

Reply:
The docs problem happens because the examples come after the first scroll, so this is why we added `position:absolute`. However, when I removed the `position:absolute` the instances at dotcom that come after the the first scroll are still the same. Seems like taking it back will fix things and not introduce any other issues as well.